db/__init__.py

```diff
@@ -70,4 +64,17 @@ def get_transaction():
     return DB.begin()
 
 
-DB = _create_engine()
+def _create_engine():
+    uri = get_config_uri()
+
+    engine = sql.create_engine(
+        uri,
+        # Set echo=True for full query logging:
+        echo=False,
+    )
+
+    return engine
+
+
+DB       = _create_engine()
+FLASK_DB = flask_sqlalchemy.SQLAlchemy()
```

Comment:
The Flask-SQLAlchemy extension is apparently a way to integrate SQLAlchemy connections in the flask request-response cycle. 

So far, I've been doing it myself by setting a new database session in the beginning of a request and clearing it afterwards. Unfortunately, this caused some problems with flask-admin. Going forward, I'll keep the same session API, but I'll let the `FLASK_DB` object provide sessions.

The existing `DB` object which encapsulates an SQLAlchemy [Engine](https://docs.sqlalchemy.org/en/20/core/connections.html#sqlalchemy.engine.Engine) is still here for direct access outside of flask (e.g. tests and scripts)